src/Microsoft.TemplateEngine.TemplateLocalizer/Commands/Export/ExportCommand.cs

```diff
@@ -89,10 +89,8 @@ protected override async Task<int> ExecuteAsync(ExportCommandArgs args, Cancella
 
             foreach (string templateJsonPath in templateJsonFiles)
             {
-                string templateDirectory = Path.GetDirectoryName(templateJsonPath) ?? string.Empty;
-                string targetDirectory = Path.Combine(templateDirectory, "localize");
 
-                ExportOptions exportOptions = new (args.DryRun, targetDirectory, args.Languages);
+                ExportOptions exportOptions = new (args.DryRun, targetDirectory: null, args.Languages);
```

Comment:
should targetDirectory be removed instead>?

Reply:
My intention is to improve the readability here. `args.DryRun` and `args.Languages` tell you the meaning of the parameter, but it is hard to understand what that `null` is for.

Reply:
I meant should we remove `targetDirectory` from ExportOptions as we are not using it? Or we are using it in certain scenarios?

Reply:
Ah, I see. Console project doesn't use it, but I added it to the API so that it can be used by anyone consuming the core library.